### PR TITLE
Updated Readme and created pull request template(#2r6cnde)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+### Screenshots of Visual Changes before/after (If There Are Any)
+<!-- If you made any changes in the UI layer, please provide before/after screenshots -->
+
+
+### Contribution and Currently Important Rules Acceptance
+<!-- Please get familiar with following info -->
+
+- [ ] I read and followed [contribution rules](https://github.com/hotwax/oms-api/blob/master/CONTRIBUTING.md)
```
Comment (ymaheshwari1) [inline .github/PULL_REQUEST_TEMPLATE.md:17]:
```suggestion
- [ ] I read and followed [contribution rules](https://github.com/hotwax/oms-api/blob/main/CONTRIBUTING.md)
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (azkyakhan) [reply .github/PULL_REQUEST_TEMPLATE.md]: I have made the suugested change.

File: README.md [hunk 2]
```diff
@@ -1 +1,94 @@
...
\ No newline at end of file
+# oms-api
+
+# Prerequisite
+Ionic CLI - If you don't have the ionic CLI installed refer [official documentation](https://ionicframework.com/docs/intro/cli) for the installation instructions.
+
+
+# Build Notes (Users)
+
+1. Download the app from [release](https://github.com/hotwax/oms-api/releases) page and extract it.
```
Comment (ymaheshwari1) [inline README.md:9]:
```suggestion
1. Download the package from [release](https://github.com/hotwax/oms-api/releases) page and extract it.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,94 @@
...
+2. Go to the app directory.
```
Comment (ymaheshwari1) [inline README.md:10]:
```suggestion
2. Go to the package directory.
```

File: README.md [hunk 4]
```diff
@@ -1 +1,94 @@
...
+3. Run following command to download dependencies  
+    `npm i
```
Comment (ymaheshwari1) [inline README.md:12]:
```suggestion
    `npm i`
```

File: README.md [hunk 5]
```diff
@@ -1 +1,94 @@
...
+4. Create a `.env` file by taking reference from the `.env.example`.
```
Comment (ymaheshwari1) [inline README.md:13]: This line is not needed.

File: README.md [hunk 6]
```diff
@@ -1 +1,94 @@
...
+5. To run the app in browser use the command: `ionic serve`
```
Comment (ymaheshwari1) [inline README.md:14]:
```suggestion
4. Build the package using `npm run build`
```

File: README.md [hunk 7]
```diff
@@ -1 +1,94 @@
...
+
+1. Open a Terminal window
+2. Clone app using the command: `git clone https://github.com/hotwax/oms-api.git <repository-name>`
+3. Go to the <repository-name> directory using command: `cd <repository-name>`
+4. Run following command to download dependencies
+    `npm i`
+5. Create a `.env` file by taking reference from the `.env.example`.
+6. To run the app in browser use the command: `ionic serve`
+
+# Firebase Hosting
```
Comment (ymaheshwari1) [inline README.md:27]: Remove this heading, sub-headings and it's content.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (ymaheshwari1) [reply README.md:14]: Do the same changes for the build notes (contributors) as well.

File: README.md (hunk 2 above)
Comment (azkyakhan) [reply README.md]: I have replaced `app` with `package`.

File: README.md (hunk 3 above)
Comment (azkyakhan) [reply README.md]: I have replaced `app` with `package`.

File: README.md (hunk 5 above)
Comment (azkyakhan) [reply README.md]: Okay I have removed it.

File: README.md (hunk 6 above)
Comment (azkyakhan) [reply README.md]: I have made the suggested change.

File: README.md (hunk 7 above)
Comment (azkyakhan) [reply README.md]: Okay I have removed it.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**